src/extension.ts

```diff
@@ -62,17 +62,44 @@ export function activate(context: vscode.ExtensionContext) {
         arrResult.push({ statement: current, block: [] });
       }
     }
-
-    console.log(arrResult);
     arrResult.forEach((element) => {
       vscode.window.showInformationMessage("found elements: " + element.statement);
     });
 
+    //rewrite file
+    const fd = fs.openSync(editor.document.fileName, "w");
+    let filePos = 0;
+    writeBlocks(fd, filePos, arrResult);
+    fs.closeSync(fd);
+
     context.subscriptions.push(disposable);
   });
 
 }
 
+function writeBlocks(fileDescriptor:  any, filePos: number, blockArray : any){
+  if (blockArray === undefined){
+    return 0;
+  }
+
```

Comment:
LGTM! 
Note: in future developments we could improve this  function to use entirely the loop block